SwiftGenStrings/SwiftTokenizer.swift

```diff
@@ -79,3 +103,17 @@ class SwiftTokenizer {
     }
 
 }
+
+private extension CharacterIterator {
+    
+    func skipWhitespace() {
```

Comment:
nit: This method could be moved to CharacterIterator and made "public" (internal).. There are some tests in CharacterIteratorTests, this method could benefit from a basic test as well